code_to_gast/helpers/js_helpers.py

```diff
@@ -3,7 +3,7 @@
 """
 handles arrays and recursively calls node_to_gast on all its elements
 """
-def js_array_expression(node):
+def js_array_expression_to_gast(node):
```

Comment:
While you're here... I'd remove the `js_` prefix from these functions. 

Definitely counter to what I usually suggest but I think all call sites should look like `js_helpers.array_expression_to_gast` so the language won't be ambiguous.

Reply:
Sounds good, just update that. We had some inconsistencies because we used camel case in the JSON object and then snake everywhere else